examples/pod_paged.rs

```diff
@@ -0,0 +1,49 @@
+use k8s_openapi::api::core::v1::Pod;
+use kube::{
+    api::{Api, ListParams, ResourceExt},
+    Client,
+};
+use tracing::*;
+
+const PAGE_SIZE: u32 = 5;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    tracing_subscriber::fmt::init();
+    let client = Client::try_default().await?;
+    let api = Api::<Pod>::default_namespaced(client);
+
+    let mut continue_token: Option<String> = None;
+    let mut page: u32 = 0;
+
+    loop {
+        page = page + 1;
+
+        info!("Fetching Page #{}", page);
```

Comment:
```suggestion
        info!("Fetching Page #{page}");
```

This could also use structured logging if desired, but either would be fine IMO:

```suggestion
        info!(page, "Fetching next page");
```